api/src/indigo_loaders.h

```diff
@@ -24,6 +24,8 @@
 #include "base_cpp/properties_map.h"
 #include "molecule/molecule.h"
 #include "reaction/reaction.h"
+#include "molecule/query_molecule.h"
+#include "third_party/rapidjson/document.h"
```

Comment:
It's better to provide `third_party` folder in includes using project's settings, and leave only 
```
#include "rapidjson/document.h"
```
or even better (since this is third-party libraries):
```
#include <rapidjson/document.h>
```
This would make the code independent from possible further changes in approach to attach third-party libs to the project - you would need to change it in one place.